src/sentry/mediators/service_hooks/creator.py

```diff
@@ -3,10 +3,25 @@
 import six
 
 from collections import Iterable
+from itertools import chain
 
 from sentry.mediators import Mediator, Param
 from sentry.models import ServiceHook
 
+# Subscribing to these events via the UI is done in a resource-centric way.
+# This means you subscribe to "Issue" events. There are many types of Issue
+# events - this maps those resource-centric values to the actual events
+# emitted.
+EVENT_EXPANSION = {
+    'issue': ['issue.created'],
```

Comment:
When the UI passes `issue` the app will expand that out to what's in this list. Same for whatever other event types end up in this dict.

Reply:
How will that work when we add 'new' events under a resource? Will existing applications get updated via migrations or will they retain the old expanded values?

Reply:
If these are ~= permissions :+1: to not magically backfilling.